src/xcode/ENA/ENA/Source/Services/CoronaTestService/__tests__/MockFamilyMemberCoronaTestService.swift

```diff
@@ -0,0 +1,157 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import OpenCombine
+import UIKit
+@testable import ENA
+
+class MockFamilyMemberCoronaTestService: FamilyMemberCoronaTestServiceProviding {
+
+	// MARK: - Init
+
+	init() {
```

Comment:
Always ask myself if this is still needed when its empty

Reply:
Seems like it's not needed in this case, removed it